### ISSUE TITLE
Latest version breaks bulma.io

### ISSUE THREAD

**Issue by stevensacks** [michaelwayman/node-sass-chokidar#43]
Create React App project.

I upgraded from 0.0.3 and it doesn't work with bulma.io.

The console is filled with errors like this:
```
{
  "status": 1,
  "file": "/Users/steven/Development/project/node_modules/bulma/sass/layout/footer.sass",
  "line": 1,
  "column": 27,
  "message": "Undefined variable: \"$background\".",
  "formatted": "Error: Undefined variable: \"$background\".\n        on line 1 of node_modules/bulma/sass/layout/footer.sass\n>> $footer-background-color: $background !default;\n   --------------------------^\n"
}
```

All I'm doing is

```
@import 'bulma/bulma.sass';
```

**Comment by stevensacks** [comment]
It changed all the bulma files in node_modules.

```
=> changed: /Users/steven/Development/project/node_modules/bulma/sass/grid/columns.sass
=> changed: /Users/steven/Development/project/node_modules/bulma/bulma.sass
=> changed: /Users/steven/Development/project/src/css/style.scss
=> changed: /Users/steven/Development/project/node_modules/bulma/sass/grid/tiles.sass
=> changed: /Users/steven/Development/project/node_modules/bulma/bulma.sass
```
etc.

**Comment by stevensacks** [comment]
It breaks Jest, too.

**Comment by michaelwayman** [comment]
I just released a new version `node-sass-chokidar@1.1.2` let me know if this fixes the issue.

**Comment by stevensacks** [comment]
Looks like it fixed it. Thanks!
